packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -88,14 +95,23 @@ export function SquadHeaderBar({
           memberCount={memberCount}
         />
       )}
-      <SimpleTooltip placement="top" content="Feedback">
+      <SimpleTooltip
```

Comment:
Totally fine for now, just noting down if this file is getting bloated maybe make a new component for this particular item. (non blocking/non urgent)